src/nsupdate/main/templates/main/includes/tabbed_router_configuration.html

```diff
@@ -281,19 +281,29 @@ <h4>inadyn ({% trans "verified with" %} 1.99.11)</h4>
 
 # note: no IPv6 support in inadyn yet.
 </pre>
-        <h4>inadyn ({% trans "verified with" %} 2.1)</h4>
+        <h4>inadyn ({% trans "verified with" %} 2.10.0)</h4>
         <pre># /etc/inadyn.conf
 
 period       = 300
 
-provider ipv4@nsupdate.info {
+# use either ipv4 or ipv6 configuration below
+#provider ipv4@nsupdate.info {
+#    ssl      = true
+#    username = {{ host.get_fqdn|default:"&lt;your hostname&gt;" }}
+#    password = {{ update_secret|default:"&lt;your secret&gt;" }}
+#    hostname = {{ host.get_fqdn|default:"&lt;your hostname&gt;" }}
+#}
+
+# IPv6 supported in inadyn, but no pre-defined provider
```

Comment:
can you raise an issue there about this? it's a bit unpretty to have such an "asymetric" way to configure the same service.

Reply:
Looking at how to resolve the problem in inadyn, I found out that a provider definition for ipv6@nsupdate.info is already present, but not mentioned in the man page. I did raise a bug report for this. I will need to adapt the config generation slightly.